spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -155,88 +138,24 @@ public Sampler defaultTraceSampler(SamplerProperties config) {
 	}
 
 	@Configuration
-	@ConditionalOnProperty(name = "spring.sleuth.http.enabled", havingValue = "true", matchIfMissing = true)
+	@ConditionalOnProperty(name = "spring.sleuth.http.enabled",
+			havingValue = "true", matchIfMissing = true)
 	@AutoConfigureBefore(TraceHttpAutoConfiguration.class)
 	public static class StackdriverTraceHttpAutoconfiguration {
```

Comment:
Are these client and server parsers still needed? If so, why aren't they part of the Sender lib?

Reply:
These parsers are "enriching" the spans, as in, adding information to them like a better name (localhost:8080 instead of no host name), some headers, etc.
They are still required, although I don't know why they aren't in the new zipkin libs...